push_notification/apps/notification/views/notification.py

```diff
@@ -0,0 +1,36 @@
+from rest_framework import status
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from apps.notification.services.notification_service import NotificationService
+from ..exceptions import NotificationNotFound
+from ..serializers.notification import NotificationSerializer
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class NotificationView(APIView):
+
+    def get(self, request):
+        logger.info("NotificationView:POST: Get all notification")
+        notifications = NotificationService().get_all_notification()
+        return Response(NotificationSerializer(notifications, many=True).data, status=status.HTTP_200_OK)
+
+    def post(self, request):
+        logger.info(msg="NotificationView:POST: payload:{}".format(request.data))
+        serialized_data = NotificationSerializer(data=request.data)
+        serialized_data.is_valid(raise_exception=True)
+        notification = NotificationService().create(serialized_data.validated_data)
+        return Response(NotificationSerializer(notification).data, status=status.HTTP_201_CREATED)
+
+    def delete(self, request):
+        notification_id = request.data.get('notification_id', None)
```

Comment:
In terms of REST principles, this should take a notification id as a parameter in the url. 
`DELETE /api/v1/notifications/:id`

Reply:
Noted